src/utilities/checkArticleExists.js

```diff
@@ -0,0 +1,34 @@
+import models from '../models';
+
+const { Article, Users } = models;
+
+/**
+ *
+ * @function checkArticleExists
+ *
+   *@description checks if article exists on the database
+
+   * @param {string} articleId - id of article
+   *
+   * @returns {object} - message from server
+   */
+
+const checkArticleExists = async (articleId) => {
+  try {
+    const article = await Article
+      .findByPk(
+        articleId,
+        {
+          include:
+          {
+            model: Users,
+            attributes: ['email']
+          }
+        }
+      );
+    return article;
```

Comment:
I return the article because it's used in a different file. The function is not intended to send a response to the user. This way, anytime the function is invoked, it either returns an article (if it exists) or nothing (if the article doesn't exist). If I want to retrieve anything from the existing article, I also can.  Also, `not good enough` is not constructive feedback, by the way. That's my feedback on your feedback. :(